improver_tests/nowcasting/forecasting/test_CreateExtrapolationForecast.py

```diff
@@ -228,7 +228,7 @@ def test_with_orographic_enhancement(self):
         self.assertEqual(result.coord("forecast_period").units, "seconds")
         self.assertEqual(
             result.coord("forecast_reference_time").points,
-            self.precip_cube.coord("forecast_reference_time").points)
+            self.precip_cube.coord("time").points)
```

Comment:
As mentioned in the plugin, I'm not sure why we are using time as the source for the frt, so once I understand that I'm sure this will come clear.

Reply:
See above - happy to discuss next week if not clear.